### DISCUSSION TITLE
Spotify Integration

### DISCUSSION THREAD

**Discussion by aspleenic** [mattermost/community#13] [Fun Ideas]
Ever wonder what your Mattermost mates are listening to? Let's integrate a playlist to see what the last track anyone played was. We start with Spotify but can then add AppleMusic, Deezer, etc...

**Comment by larkox** [comment]
This idea has been around for a while, I think. It would be great to add also spotify controllers on Mattermost, so you can play, stop, or jump to the next song from within Mattermost.

**Reply by aspleenic** [reply]
I will list this for every hackathon til it happens, Daniel :)

**Comment by chetanyakan** [comment]
Could be like discord where we use profile popover component
![image](https://user-images.githubusercontent.com/35728906/123239672-3b33d880-d4fd-11eb-9896-099d621a1d71.png)

**Comment by Willyfrog** [comment]
as an extra it would be great a `Share to channel`, that would make easier to talk about a specific song
